lib/routes/nbt/sjxy/index.ts

```diff
@@ -0,0 +1,72 @@
+import { Route } from '@/types';
+import cache from '@/utils/cache';
+import { load } from 'cheerio';
+import got from '@/utils/got';
+
+const host = 'https://sjxy.nbt.edu.cn/';
+const typeMap = {
+    notice: '/index/tzgg.htm',
+    news: '/index/xwdt.htm',
+};
+
+const titleMap = {
+    notice: '通知公告',
+    news: '新闻动态',
+};
+
+export const route: Route = {
+    path: '/sjxy/:type',
+    categories: ['university'],
+    example: '/nbt/sjxy/notice',
+    parameters: { type: '通知类型，默认为通知公告' },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    name: '通知公告',
+    maintainers: ['bellongyan'],
+    handler,
+    description: `| 通知公告 | 新闻动态 |
+    | ---------- | ----- |
+    | notice     | news  |`,
+};
+
+async function handler(ctx) {
+    const type = ctx.req.param('type') ?? 'notice';
+    const link = host + typeMap[type];
```

Comment:
https://github.com/DIYgod/RSSHub/blob/66fcc61b7ff63221bfda29039dcb9f1078d729fb/lib/routes/nbt/sjxy/index.ts#L6 https://github.com/DIYgod/RSSHub/blob/66fcc61b7ff63221bfda29039dcb9f1078d729fb/lib/routes/nbt/sjxy/index.ts#L7-L10
This causes [double slashes](https://github.com/DIYgod/RSSHub/actions/runs/10753215436/job/29822393374#step:13:74) `//` in the URL.